### DISCUSSION TITLE
Trigger che compara temperatura di un sensore e input number

### DISCUSSION THREAD

**Discussion by Zagor4** [indomus/forum#515] [HUB: Home Assistant]
Ciao a tutti,
Sto provando a riscrivere una vecchia automazione e vorrei inserire questa volta i trigger "corretti" partendo dal confronto di 2 temperature: 
La prima è rilevata da un sensore di temperatura e la seconda invece è una soglia che inserisco manualmente da plancia tramite "input number".

Ho provato a scrivere questo:
```
platform: template
value_template: '{{states('sensor.temperatura_sala')|float < states('input_number.temp_low_sala')|float}}'
id: Temp_Inferiore

```

Ma mi restituisce un errore: Integration not found.
Qualche idea ?

**Comment by snakuzzo** [comment]
A parte l'utilizzo degli apici "non corretto" nel template, il problema non è questo blocco di codice, ma qualcos'altro.
Hai sbagliato a scrivere qualche altra cosa (tipo indentazione o cose simili)

P.S. Usa gli apici doppi fuori e i singoli all'interno (o viceversa)...
`value_template: "{{states('sensor.temperatura_sala') | float < states('input_number.temp_low_sala') | float}}"`

**Reply by Zagor4** [reply]
Si poi ho trovato il problema... lo so che è imbarazzante ma lo scrivo perchè magari potrebbe essere utile a qualcun altro che ha iniziato da poco... 
Utilizzando l'editor per le automazioni, avevo inserito il trigger e nelle condizioni era rimasto il tipo di condizione ma senza null'altro e quindi la condizione non risultava completa restituendo l'errore.

Grazie mille

**Reply by snakuzzo** [reply]
Consiglio...se volete usare l'editor, fatelo giusto se vi serve aiuto per la struttura, ma poi fate tutto via YAML. L'editor via GUI è spazzatura
